applications/stepswitch/src/stepswitch_inbound.erl

```diff
@@ -18,11 +18,12 @@
 %%--------------------------------------------------------------------
 -spec handle_req(wh_json:object(), wh_proplist()) -> 'ok'.
 handle_req(JObj, _Props) ->
-    'true' = wapi_route:req_v(JObj),
     _ = wh_util:put_callid(JObj),
+    'true' = wapi_route:req_v(JObj),
     case wh_json:get_ne_value(?CCV(<<"Account-ID">>), JObj) of
         'undefined' -> maybe_relay_request(JObj);
-        _AcctID -> 'ok'
+        AccountId -> lager:debug("fetch-id ~s already has account-id ~s, skipping.",
```

Comment:
@jamesaimonetti actually that was the point ;) we were missing the information during a debug session and we needed to know which apps ignored  a route_req.  Since it was a commit to 3.22 it needs to be on master as well (by policy) and I am behind that.  We can discuss removal if the logs are too chatty after we have collected the data we need :)